convert/src/vertex_to_bd/badgerdoc_format/pdf_renderer.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+import fitz
+from fitz import Page
+
+from ..models import BadgerdocToken
+
+
+class Fonts(Enum):
+    COURIER = "cour"
+
+
+class PDFRenderer:
+    def __init__(
+        self,
+        page_border_offset: int = 15,
+        font_name: str = Fonts.COURIER.value,
+        font_size: int = 11,
+    ):
+        self.page_border_offset = page_border_offset
+        self.font_name = font_name
+        self.font_size = font_size
+
+    def render_tokens(
+        self, tokens: List[BadgerdocToken], save_path: Path
+    ) -> None:
+        doc = fitz.open()
+        width = (
+            max((token.bbox[2] for token in tokens)) + self.page_border_offset
+        )
+        height = (
+            max((token.bbox[3] for token in tokens)) + self.page_border_offset
```

Comment:
You can remove redundant parenthesis in `max`

Reply:
fixed